proto/blueprint.proto

```diff
@@ -479,6 +479,71 @@ message Card {
    * Mega-boosted cards show a even larger headline size.
    */
   optional bool mega_boosted = 38;
+
+  /**
+   * Indicate how many columns the trail image is expected to take
+   */
+  optional int32 trail_image_size = 39;
+
+  /**
+   * Define constants for each image aspect ratio supported
+   */
+  enum ImageAspectRatio {
+    IMAGE_ASPECT_RATIO_UNSPECIFIED = 0;
+    IMAGE_ASPECT_RATIO_LANDSCAPE_5_3 = 1;
+    IMAGE_ASPECT_RATIO_LANDSCAPE_5_4 = 2;
+    IMAGE_ASPECT_RATIO_PORTRAIT_4_5 = 3;
+    IMAGE_ASPECT_RATIO_SQUARE = 4;
+  }
+
+  /**
+   * Indicate in what aspect ratio the trail image should be rendered
+   */
+  optional ImageAspectRatio trail_image_aspect_ratio = 40;
+
+  /**
+   * Define constants for the arrangement of sublinks on a card
+   */
+  enum SublinksArrangement {
+    SUBLINKS_ARRANGEMENT_UNSPECIFIED = 0;
```

Comment:
I'm curious - why do we need unspecified? Is this a protobuf requirement?

Reply:
It provides a way for client code to handle default/fallback values - https://github.com/guardian/mobile-apps-api-models/pull/31  

The constant which is mapped to 0 in an `enum` is the default value in protobuf format.